### PR TITLE
feature/ari 322 2

### PR DESCRIPTION
**Pull request by benj2k**
- fix(arianee-access-token): exp and nbf are in seconds not in ms
- feat(jwt): jwt exp is in second

### CODE REVIEW

**Comment by stefdelec** [comment]
DO NOT MERGE! We need to wait for the first pr to be merged && deployed before merging this one

**Review by 0xWhiteleaf — APPROVED** [review]

**Pull request merged**
